modules/subscribe/SubscribeForm.tsx

```diff
@@ -10,8 +11,8 @@ type FormData = {
   FNAME: string;
   LNAME: string;
   EMAIL: string;
-  ISNL: boolean;
-  SPONSOR: boolean;
+  ISNL: boolean | string;
```

Comment:
Had add an union of `string` to pass the data to the API as a string. Could probably clean this up better though.

Reply:
All good :)